packages/typescript-fetch-client/templates/api.hbs

```diff
@@ -174,12 +174,13 @@ export const {{className name}}ApiFetchParamCreator = function (configuration?:
 	{{/with}}
 	{{/if}}
 
-			const localVarQueryParameterString = localVarQueryParameter.toString();
-			if (localVarQueryParameterString) {
-				localVarPath += "?" + localVarQueryParameterString;
+			const url = new URL('https://example.com');
```

Comment:
Sadly, to use the new URL() API, we need to give the constructor a protocol and host. I don't like this line, and would welcome some suggestions for making it less weird 😄